genet/output/geojson.py

```diff
@@ -59,28 +59,53 @@ def line_geometry():
     return {"nodes": nodes, "links": links}
 
 
-def save_geodataframe(gdf, filename, output_dir, include_shp_files=False):
+def save_geodataframe(
```

Comment:
There's some repetition here with `write_spatial` in `core.py`. Not sure where it is best placed. Maybe `write_spatial` should defer the check for whether one of `to_...` is defined to `save_geodataframe`

Reply:
made a check method that gets called when saving but reuse it in core too (because it takes a bit of time to format the network, so you don't want to wait a while to find out the format you want is not right)